pulp_rpm/app/tasks/copy.py

```diff
@@ -55,28 +55,24 @@ def find_children_of_content(content, src_repo_version):
     for advisory in advisories:
         # Find rpms referenced by Advisories/Errata
         package_nevras = advisory.get_pkglist()
+        advisory_package_q = Q()
         for nevra in package_nevras:
             (name, epoch, version, release, arch) = nevra
-            try:
-                package = packages.get(
-                    name=name, epoch=epoch, version=version, release=release, arch=arch)
-                children.add(package.pk)
-            except Package.DoesNotExist:
-                raise
-            except MultipleObjectsReturned:
-                raise
+            advisory_package_q |= Q(
+                name=name, epoch=epoch, version=version, release=release, arch=arch
+            )
+        for package in packages.filter(advisory_package_q).iterator():
+            children.add(package.pk)
 
         module_nsvcas = advisory.get_module_list()
+        advisory_module_q = Q()
         for nsvca in module_nsvcas:
             (name, stream, version, context, arch) = nsvca
-            try:
-                module = modules.get(
-                    name=name, stream=stream, version=version, context=context, arch=arch)
-                children.add(module.pk)
-            except Modulemd.DoesNotExist:
-                raise
-            except MultipleObjectsReturned:
```

Comment:
I guess technically this case isn't being handled anymore, but since the selection of Packages / Modules is coming from only what is present in the repo, I don't think it should be an issue?

Reply:
Agreed. 
When we create a repo version we should ensure that there is only one package/module with corresponding NEVRA/NSVCA.